dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/rest/service/DssRestActorJsonService.java

```diff
@@ -75,7 +76,7 @@ protected S getBody(String content) {
             final ObjectMapper mapper = new ObjectMapper();
             return mapper.readValue(content, typeReference);
         } catch (JsonProcessingException e) {
-            return null;
+            throw new DssRestJsonRequestMappingException(e.getMessage());
```

Comment:
I think, it is better to throw new DssRestJsonRequestMappingException(e);

Reply:
Okay, I will.